website/tests/pages/review/index.spec.ts

```diff
@@ -48,4 +49,25 @@ test.describe('The review page', () => {
 
         await reviewPage.waitForTotalSequenceCountCorrect(total, 'less');
     });
+
+    test('approve restricted sequences', async ({ reviewPage, loginAsTestUser }) => {
+        const { token, groupId } = await loginAsTestUser();
+
+        await reviewPage.goto(groupId);
+
+        const { total } = await reviewPage.getReviewPageOverview();
+
+        await prepareDataToBe('awaitingApprovalRestricted', token, groupId);
+
+        await reviewPage.waitForTotalSequenceCountCorrect(total + testSequenceCount);
+
+        await reviewPage.approveAll();
+
+        await reviewPage.waitForTotalSequenceCountCorrect(total);
+
+        await reviewPage.page.goto(
+            `${baseUrl}${routes.mySequencesPage(dummyOrganism.key, groupId)}dataUseTerms=RESTRICTED`,
+        );
+        reviewPage.page.getByText(`Search returned ${testSequenceCount} sequence`);
```

Comment:
For safety, it might make sense to click on the top sequence and check that its data use terms are indeed restricted this is because:

if we rename `dataUseTerms` to e.g. `data_use_terms` but forget to change this test then the  `dataUseTerms=RESTRICTED` filter above will cease to have an effect so we could get a test that always passes even if our code stops displaying the restricted data properly

Reply:
We could do this by looking for selectors regex matching `LOC_`

Reply:
Done :-) I click the first on the first accession link in the table :-)

Reply:
Nice!